### DISCUSSION TITLE
save failed replays

### DISCUSSION THREAD

**Discussion by Hammerhead042** [ppy/osu#13561] [Ideas] [answered]
Maybe it's already possible and I just don't know how, but if it isn't, would it be possible to add that ability?
Like F1 / F2 in stable or maybe two more buttons in the fail-screen for "show replay" and "save replay".

It is really frustrating to almost pass Galaxy Collapse and having no proof (;_;)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/8721.
